pkgs/development/compilers/gcc/8/default.nix

```
@@ -65,12 +65,13 @@ let majorVersion = "8";
     /* Cross-gcc settings (build == host != target) */
     crossMingw = targetPlatform != hostPlatform && targetPlatform.libc == "msvcrt";
     stageNameAddon = if crossStageStatic then "-stage-static" else "-stage-final";
```

Let's remove the dash in all of these, while we are at it?
```suggestion
  stageNameAddon = if crossStageStatic then "stage-static" else "stage-final";
```

---

I don't understand - without the dash, the components will run together and be unreadable, no? Or are you suggesting we refactor the whole thing to put the dashes at the point of interpolation? (That would be much nicer, for sure.)

---

Ah yes, I see.